pom.xml

```diff
@@ -9,8 +9,8 @@
 	<properties>
 		<revision>8.1.0-SNAPSHOT</revision>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<maven.compiler.target>11</maven.compiler.target>
-		<maven.compiler.source>11</maven.compiler.source>
+		<maven.compiler.target>17</maven.compiler.target>
+		<maven.compiler.source>17</maven.compiler.source>
```

Comment:
Does this mean we will drop support for JDK11 already?

Reply:
This is too fast.. `compiler.source` should still be at 11, I think. I think it would be wise to move to 17 language level when we don't need to do often backports to < v8.0 as well.